src/lib/cbe/cbe-cache.ads

```diff
@@ -48,11 +48,11 @@ is
    --
    --  Data_Index
    --
-   function Data_Index (
-      Obj : in out Object_Type;
-      PBA :        Physical_Block_Address_Type;
-      Ts  :        Timestamp_Type)
-   return Cache_Index_Type;
+   procedure Data_Index (
+      Obj    : in out Object_Type;
```

Comment:
AFAICS we always align the `:` to the longset parameter, e.g.

```Ada
procedure Foo (
   First  : Type;
   Second : Type;
   Third  : Type)
```

I'm not sure if that is against the *Ada Style Guide* but in case it is not I would rather keep it that way.

Reply:
I somehow missed the `Result` paramter and thought there were only the three first ones.